pokemon-league-participation-manager/pom.xml

```diff
@@ -1,10 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>cz.muni.fi.pa165</groupId>
-    <artifactId>pokemon-league-participation-manager</artifactId>
-    <version>1.0-SNAPSHOT</version>
+
+    <parent>
+        <groupId>cz.muni.fi.pa165</groupId>
+        <artifactId>eshop-parent</artifactId>
```

Comment:
e-shop parent? don't we want some other name for parent project like pokemon-league-... ?